apps/web/src/actions/copilot/refinePrompt.ts

```diff
@@ -88,7 +88,10 @@ export const refinePromptAction = withDocument
       projectId: env.COPILOT_PROJECT_ID,
     }).then((r) => r.unwrap())
 
-    const result = await sdk.prompts.run(env.COPILOT_PROMPT_REFINE_PATH, {
```

Comment:
I published new SDK version and now we have beautiful typed returns of the SDK breaking these types. I think this fix it